docs/context.rst

```diff
@@ -0,0 +1,44 @@
+###########
+The Context
+###########
+
+Why
+===
+
+Problem
+-------
+
+Around the beginning of calendar year 2021, it was recognized within edX that the ID Verification process we
+enforce the learner though, was not aligning with the mission of wider educational access. Specific examples
+of ID verification challenges were (in no particular order):
+
+- ID Verification vendors do not support a wide range of languages. Asian students had a hard time getting verified
+- ID Verification process has a greater than 20% denial rate. This introduced a lot of support issues and learners pain
+- Before proctoring exams, we would require ID Verification. This means learners has to provide PII evidence
+  twice before exams. Furthermore, learners who has trouble with ID Verification process sometimes lost the chance to take the exams
+- edX collecting learners verification documents is at security risk of hacking and leaking.
+- The cost of ID verification process on edX is trending higher as the number of active paying learners grow
+
+
+Solution
+--------
+
+It was proposed by edX product management that we should create a newer and more lightweight process to replace
```

Comment:
A big concept that came up throughout the ideation and implementation process was the idea of trusting the learner and assuming good intent. We wanted to reduce the barrier to entry for being a leaner on edX to a minimal set of learners that want to change the name on their certificates. I think this would be useful to call out, because it drove a lot of our decision making and was a key way that we justified this work to partners that might otherwise have had integrity concerns.